app/src/main/java/org/grapheneos/apps/client/App.kt

```diff
@@ -211,12 +211,7 @@ class App : Application() {
 
         if (!isServiceRunning) {
             if (foregroundServiceNeeded && !isSeamlessUpdateRunning()) {
-                startService(
-                    Intent(
-                        this@App,
-                        KeepAppActive::class.java
-                    )
-                )
+                startService(Intent(this@App, KeepAppActive::class.java))
```

Comment:
I don't know the Kotlin official codestyle off the top of my head, but if this is going against it then it shouldn't be changed here.

Reply:
It should only apply for code exceeding 100 columns, and this one is nowhere near 100, and is not being warned in Android Studio

Reply:
For example: https://github.com/GrapheneOS/Apps/blob/a58545e031ff00b4b3184aacab6e433a3e9afba7/app/src/main/java/org/grapheneos/apps/client/App.kt#L1012

https://github.com/GrapheneOS/Apps/blob/a58545e031ff00b4b3184aacab6e433a3e9afba7/app/src/main/java/org/grapheneos/apps/client/App.kt#L214

Reply:
Ah okay, was just checking. My eyes couldn't tell if it was over 100 or not lol.